test/sync-controller.test.js

```diff
@@ -226,6 +229,50 @@ QUnit.test('uses separate date time to display time mapping for each timeline',
   );
 });
 
+QUnit.test('ProgramDateTime strategy finds nearest llhls sync point', function(assert) {
+  const strategy = getStrategy('ProgramDateTime');
+  const playlist = playlistWithDuration(200, {llhls: true});
+  const timeline = 0;
+  const duration = Infinity;
+  let syncPoint;
+
+  syncPoint = strategy.run(this.syncController, playlist, duration, timeline, 170);
+
+  assert.equal(syncPoint, null, 'no syncpoint when no date time to display time mapping');
+
+  playlist.segments.forEach((segment, index) => {
+    segment.dateTimeObject = new Date(2012, 11, 12, 12, 12, 12 + (index * 10));
+  });
+
+  this.syncController.setDateTimeMappingForStart(playlist);
+
+  const newPlaylist = playlistWithDuration(200, {llhls: true});
```

Comment:
Any reason the new playlist is created versus using the old?

Reply:
not entirely sure, I updated this test to just use the old playlist.